src/widgets/copilot/components/Messages/LocationModal.tsx

```
@@ -0,0 +1,522 @@
+        googleMapRef.current.setZoom(15);
+      } else if (leafletMapRef.current) {
+        leafletMapRef.current.flyTo(
+          [newLocation.latitude, newLocation.longitude],
+          15,
+          { animate: true, duration: 0.5 },
+        );
+      } else {
+        console.warn("Leaflet map is not fully initialized yet.");
+      }
+    },
+    (_) => {
```

ESLint

[error] '_' is defined but never used.

(@typescript-eslint/no-unused-vars)